Updates to billing FAQs (#3911)
dbt-labs/docs.getdbt.com: website/docs/docs/cloud/billing.md

dbt Cloud considers a Successful Model Built as any model that is successfully built via a run through dbt Cloud’s orchestration functionality in a dbt Cloud deployment environment. Models are counted when built and run. This includes any jobs run via dbt Cloud's scheduler, CI builds (jobs triggered by pull requests), runs kicked off via the dbt Cloud API, and any successor dbt Cloud tools with similar functionality. This also includes models that are successfully built even when a run may fail to complete. For example, you may have a job that contains 100 models and on one of its runs, 51 models are successfully built and then the job fails. In this situation, only 51 models would be counted.

> Any models built in a dbt Cloud development environment (for example, via the IDE) do not count towards your usage. Tests, seeds, ephemeral models, and snapshots also do not count. 

Reviewer: ```suggestion
Any models built in a dbt Cloud development environment (for example, with the IDE) don't count towards your usage. Tests, seeds, ephemeral models, and snapshots also don't count. 
```
